### DISCUSSION TITLE
erro no navbar - Unexpected token / at column 1 in [/account]

### DISCUSSION THREAD

**Discussion by silvarafaell** [balta-io/7187#39] [Perguntas e Respostas] [answered]
Boa noite, estou com problemas muitos parecidas no navbar e algumas pages. 
![erro navbar](https://user-images.githubusercontent.com/22201401/129278063-85ce49bb-8855-4b1d-9ae8-82c8aa6c8a33.png)
![error 1 navbar](https://user-images.githubusercontent.com/22201401/129278075-53eef9fd-2f3c-4804-b8c5-9c89a01d4056.png)
![error 2 navbar](https://user-images.githubusercontent.com/22201401/129278114-63ed1a2a-b8d5-42d5-ba9c-27ab85f11417.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @silvarafaell, tentou executar o código do curso para ver se não digitou nada errado?
Em adicional, está usando as mesmas versões que as do curso?
